astropy/nddata/utils.py

```diff
@@ -633,11 +633,11 @@ def __init__(self, data, position, shape=None, side_length=None, wcs=None,
         self.input_position_original = position
         self.shape_input = shape
 
```

Comment:
You're right, that was my mistake in the comment or maybe that was before we switched to the numpy indexing convention. Thanks!